### PR TITLE
2677/tags in sms

### PR DESCRIPTION
**Pull request by cornelinux**

### CODE REVIEW

**Review by plettich — COMMENTED** [review]
Looks good, just a short comment

File: doc/policies/authentication.rst [hunk 1]
```diff
@@ -161,6 +162,22 @@ Starting with version 2.20 you can use the tag *{challenge}*. This will add
...
+  * {givenname} the given name of the token owner.
+  * {surname} the surname of the token owner.
+  * {username} the loginname of the token owner.
+  * {userrealm} the realm of the token owner.
+  * {tokentype} the type of the token.
+  * {recipient_givenname} the given name of the recipient.
+  * {recipient_surname} the surname of the recipient.
+  * {time} the current server time in the format HH:MM:SS.
+  * {date} the current server date in the format YYYY-MM-DD
+
```
[the remark this answers is not in the document]
Comment (cornelinux) [reply doc/policies/authentication.rst:180]: haha. I also thought about that email allows to add a `file:/...` to contain the template. But well, due to the SHORT message system this did not seem sensible to me :-)

I think the hint that the gateway might fail with too long messages is a good idea!

[the remark this answers is not in the document]
Comment (cornelinux) [reply doc/policies/authentication.rst:180]: done in d37bef44

File: doc/policies/authentication.rst [hunk 2]
```diff
@@ -161,11 +162,32 @@ Starting with version 2.20 you can use the tag *{challenge}*. This will add
...
 In the :ref:`sms_gateway_config` the tag *{otp}* will be replaced by the custom
 message, set with this policy.
 
 Default: *<otp>*
 
+.. note:: The length of an SMS is limited to 140 characters due to the definition of SMS.
+   You should take care, that the *smstext* does not exceed this limit. SMS gateways could
+   reject to long messages or the delivery could fail.
```
Comment (plettich) [inline doc/policies/authentication.rst:188]: ... reject *too* long ...

**Review by plettich — CHANGES_REQUESTED** [review]
Looks good, just a minor spelling issue

File: doc/policies/authentication.rst (hunk 2 above)
Comment (cornelinux) [reply doc/policies/authentication.rst]: Done in 40e2f5f3

**Review by plettich — APPROVED** [review]
Looks good

**Pull request merged**